proto/viam/app/data/v1/data.proto

```diff
@@ -28,14 +28,14 @@ service DataService {
   // DeleteBinaryDataByIDs deletes binary data based on given IDs.
   rpc DeleteBinaryDataByIDs(DeleteBinaryDataByIDsRequest) returns (DeleteBinaryDataByIDsResponse);
 
-  // AddTagsToBinaryDataByFileIDs adds string tags, unless the tags are already present, to binary data based on given IDs.
-  rpc AddTagsToBinaryDataByFileIDs(AddTagsToBinaryDataByFileIDsRequest) returns (AddTagsToBinaryDataByFileIDsResponse);
+  // AddTagsToBinaryDataByIDs adds string tags, unless the tags are already present, to binary data based on given IDs.
```

Comment:
These Add/Remove tags APIs are only used in app (which is fixed in the corresponding app PR), not in RDK, and not directly via proto by any existing users.